src/sparseml/transformers/sparsification/modification/base.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from packaging import version
+
+from sparseml.base import get_version
+
+
+_LOGGER = logging.getLogger(__name__)
+
+__all__ = ["check_transformers_version"]
+
+_TRANSFORMERS_MIN_VERSION = "4.36.0"
+_TRANSFORMERS_MAX_VERSION = "4.37.0"
```

Comment:
I'm okay with this as long as it is a warning, even if it is annoying to maintain. Speaking of which, we should probably change the torch max version to a warning rather than a failure

Reply:
@mgoin regarding the maintenance -> how bout we just parse the accepted transformers range from `setup.py` (simple regex on the `cat` of this file), instead of returning to this script and manually editing the range?

Reply:
@mgoin it's already emitting a warning

Reply:
Cool, thank you